apps/web/app/pages/stacking/components/pool-overview.tsx

```diff
@@ -0,0 +1,142 @@
+import { css } from 'leather-styles/css';
+import { Box, VStack, styled } from 'leather-styles/jsx';
+import { DummyIcon } from '~/components/dummy';
+import { InfoGrid } from '~/components/info-grid/info-grid';
+import { Pool } from '~/features/stacking/start-pooled-stacking/utils/types-preset-pools';
+import { ValueDisplayer } from '~/pages/sbtc-rewards/components/reward-value-displayer';
```

Comment:
Please can we move this into the components dir so we don't cross import from features